microprot/scripts/process_fasta.py

```diff
@@ -123,11 +124,53 @@ def read_representatives(represent):
     return sorted(list(ids))
 
 
+def split_fasta(seqs, prefix=None, outdir=None):
```

Comment:
I have the feeling that we better use skbio methods to perform the FASTA parsing. Splitting should be easy than, by just writing each sequence object from the generator into a file.
Parsing FASTA can be tricky sometimes, if you consider multi-line sequences, empty but not separating lines, comment lines, variable line length, ...